.eslintrc

```diff
@@ -14,19 +15,17 @@
   },
   "plugins": [
     "react",
-    "mocha"
+    "jest"
   ],
   "env": {
     "node": true,
     "browser": true,
-    "es6": true,
-    "mocha": true
+    "es6": true
   },
   "rules": {
     "import/no-extraneous-dependencies": 0,
     "react/forbid-prop-types": [2, { "forbid": ["any", "array"] }],
     "react/require-default-props": 0,
-    "mocha/no-exclusive-tests": 2,
```

Comment:
any similar rules for jest?

Reply:
Yeah we got a bunch from the added jest plugin, in fact even more 🎉